templates/home.html

```diff
@@ -609,6 +646,122 @@ <h5 class="mb-1" id="title"></h5>
+                    let requestDescription = sourceRequest.find("#description");
+                    let requestFooter = sourceRequest.find("#requestFooter");
+                    let requestContentWrapper = $("<div></div>")
+                        .attr("id", "contentWrapper" + selectedRequestId)
+                        .addClass("collapse")
+                        .append(requestDescription)
+                        .append(requestFooter);
+                    sourceRequest.append(requestContentWrapper);
+
+                    let listElem = $("<li></li>")
+                        .addClass("d-flex w-100 justify-content-between");
+
+                    let contentContainer = $("<div></div>")
+                        .addClass("w-100")
+                        .append(sourceRequest);
+
+                    listElem.append(contentContainer);
+
+                    let buttonContainer = $("<div></div>")
+                        .addClass("d-flex flex-nowrap ml-auto");
+
+                    if (selectedRequests.length > 1) {
+                        let mergeButton = $("<button></button>")
+                            .attr("type", "button")
+                            .click({ index: i }, mergeSelectedTo)
+                            .text("Merge to this")
+                            .addClass("btn btn-warning btn-sm ml-2")
+                            .css("height", "38px");
+
+                        buttonContainer.append(mergeButton);
+                    }
+
+                    let removeButton = $("<button></button>")
+                        .attr("type", "button")
+                        .click({ index: i }, removeRequestFromList)
+                        .text("X")
+                        .addClass("btn btn-outline-danger btn-sm ml-2")
+                        .css("height", "38px");
+
+                    buttonContainer.append(removeButton);
+
+                    listElem.append(buttonContainer);
+
+                    modalList.append(listElem);
+                });
+            }
+
+            function addRequestToList(sender) {
+                let requestId = sender.attributes["requestId"].value;
+                if (selectedRequests && !selectedRequests.some(sr => sr == requestId))
+                    selectedRequests.push(requestId);
+                setMergeModalButtonVisibility();
+            }
+
+            function removeRequestFromList(event) {
+                selectedRequests.splice(event.data.index, 1);
+                refreshMergeModalList();
+                setMergeModalButtonVisibility();
+                if (selectedRequests.length < 1) {
+                    $("#mergeModal").modal('hide');
+                }
+            }
+
+            function mergeSelectedTo(event) {
+                let targetIndex = event.data.index;
+                let targetId = selectedRequests.splice(targetIndex, 1)[0];
+                let payload = { targetId: targetId, sourceIds: selectedRequests }

```

Comment:
```suggestion
                let payload = { id: targetId, ids_from: selectedRequests }

```

For the sake of consistency with earlier backend code. I'm gonna have to adjust the backend later to parse json (required in this case because arrays, as opposed to all other endpoints) and handle the array, but ill do that when everything else is done.

Reply:
changed payload to your suggestion